jhove-modules/src/main/java/edu/harvard/hul/ois/jhove/module/pdf/Parser.java

```diff
@@ -96,7 +94,7 @@ public Token getNext (long max) throws IOException, PdfException
         else if (tok instanceof DictionaryEnd) {
             --_dictDepth;
             if (_dictDepth < 0) {
-                throw new PdfMalformedException (MessageConstants.ERR_DICT_DELIMETERS_IMPROPERLY_NESTED);
+                throw new PdfMalformedException (MessageConstants.ERR_DICT_DELIMITERS_IMPROPERLY_NESTED);
```

Comment:
PDF-HUL-33

Reply:
Good catch, am thinking of adding a consistency check for errors in unit tests, but this is why adding an error number to the message soon is a good idea. Then the compiler takes care of it.